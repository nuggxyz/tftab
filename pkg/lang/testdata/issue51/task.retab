

gen out {
    path = "out/file.yaml"
    data = runner(1)
}


func runner {
	params = [ref]
	result = {
		a = ref,
		b = 2,
		c = 3,
		d = 4,
		e = 5,
		f = 6,
		g = 7,
		h = 8,
		i = 9,
	}
}
