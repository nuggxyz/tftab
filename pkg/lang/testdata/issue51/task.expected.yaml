# code generated by retab v0.0.0-unknown. DO NOT EDIT.
# join the fight against yaml @ github.com/walteh/retab

# source: "task.retab"

a: 1
b: 2
c: 3
d: 4
e: 5
f: 6
g: 7
h: 8
i: 9
